providers/implementations/ciphers/cipher_aes_ccm_hw_rv32i_zknd_zkne.inc

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*-
+ * RISC-V 32 ZKND ZKNE support for AES CCM.
+ * This file is included by cipher_aes_ccm_hw.c
+ */
+
+static int ccm_rv32i_zknd_zkne_initkey(PROV_CCM_CTX *ctx, const unsigned char *key,
+                             size_t keylen)
+{
+    PROV_AES_CCM_CTX *actx = (PROV_AES_CCM_CTX *)ctx;
+
```

Comment:
Spurious blank line.

Reply:
https://github.com/openssl/openssl/blob/56233ba8574c01b3912cf662335fedaabc7faec2/providers/implementations/ciphers/cipher_aes_ccm_hw_aesni.inc#L18-L20

This is copy-pasted from another file.